src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -14180,6 +14356,41 @@ Label900: ;
 		return StringOut;
 	}
 
+	bool 
+	isNumber(std::string s)
+	{
+		bool is = true;
+		try {
+			std::stof(s);
+		} catch ( std::exception & e ) {
```

Comment:
If an exception is rarely thrown, then it would be similar to (and maybe just slightly faster) than an if statement. However, when an exception is thrown there is a large cost associated with it. So the big question is how often this function is checking actual numbers versus non-numbers. If it isn't overwhelmingly numbers then you'd have to profile to see the actual performance impact.

I understand this is reporting code and is only run once. I was on the fence about suggesting a change to begin with because of that reason. I just know how programming patterns have a tendency to move around in E+ and I don't want this pattern in the simulation code itself eventually.

Also if @JasonGlazer sticks with the exception based approach then it should change to `std::stod` like I mentioned in my original post.